src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -11986,6 +11986,11 @@ void WriteVeriSumTable(EnergyPlusData &state)
                         totPlugProcess += state.dataHeatBal->ZoneHWEq(iPlugProc).DesignLevel;
                     }
                 }
+                for (iPlugProc = 1; iPlugProc <= state.dataHeatBal->TotITEquip; ++iPlugProc) {
+                    if (iZone == state.dataHeatBal->ZoneITEq(iPlugProc).ZonePtr) {
+                        totPlugProcess += state.dataHeatBal->ZoneITEq(iPlugProc).DesignTotalPower;
+                    }
+                }
```

Comment:
Add ITE total design power to sum of zone plug loads.

Reply:
Seems reasonable to me.

Reply:
for (auto &zoneITEq : state.dataHeatBal->ZoneITEq)

Reply:
Definitely right on that @rraustad; I wasn't going to push on that for this quick last minute fix, but if this is delayed for any reason it should be cleaned up.